File: game.cpp
// A simple "Guess The Number" game made using C++

#include<bits/stdc++.h>

using namespace std;

int main()
{
    srand(time(0));
    int guess, number, choice;
    number = (rand() % 200) + 1;
    cout << "\nI am thinking of a number! Try to guess that number (Hint: The number's between 1 and 200): ";
    cin >> guess;
    while(guess != number)
    {
        if(guess < number)
        {
            cout << "\n\nToo low! Guess again: ";
            cin >> guess;
        }
        else if(guess > number)
        {
            cout << "\n\nToo high! Guess again: ";
            cin >> guess;
        }
    }
    cout << "\n\nThat's it! You have guessed the number correctly :)\n";
    cout << "\n\nWould you like to play again? (Enter 1 for 'Yes'): ";
    cin >> choice;
    if(choice  == 1)
        return main();
    return 0;
}

File: game.java
import java.util.*;
import java.io.*;

public class game{
    public static void main(String [] args){
        int num;
        String cond;
        Random rand = new Random();
        Scanner input = new Scanner(System.in);
        Scanner scanner = new Scanner(System.in);
        int sec = rand.nextInt(200) + 1;
        
        System.out.println("I'm thinking of a number! Try to guess the number I'm thinking of: ");
    	int num = input.nextInt();    
        while(num!=sec) {
            if(sec>num) {
                System.out.println("Too low! Guess again: ");
    	        int num = input.nextInt();    
            }
            else if(sec<num) {
                System.out.println("Too high! Guess again: ");
    	        int num = input.nextInt();    
            }
        }
        System.out.println("That's it! Would you like to play again? (yes/no): ");
    	String cond = scanner.nextLine();
        if((cond == "yes") || (cond == "Yes") || (cond == "YES") ) {
            //return main();
        }
        else {
            //return 0;
        }
    }
}

